demo-app/src/components/atoms/Button/index.tsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import {Button} from '@mui/material';
+interface ButtonProps{
+    children:React.ReactNode;
+    style:React.CSSProperties;
+    onClick:()=>void;
+}
+const ButtonComp:React.FC<ButtonProps>=(props)=>{
```

Comment:
You can destructure the props here itself

eg: const ButtonComp:React.FC<ButtonProps>=(children,style,onClick)=>{

Reply:
updated